test/performance/resources/templates/deployment-plc.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-dep-[ID]
+  namespace: default
+  annotations:
+    policy.open-cluster-management.io/categories: CM Configuration Management
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+    policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+  labels:
+    grc-test: config-policy-performance
+spec:
+  disabled: false
+  remediationAction: enforce
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: dep-create-[ID]
+        spec:
+          remediationAction: enforce
+          severity: low
+          namespaceSelector:
+            exclude:
+              - kube-*
+            include:
+              - default
+          pruneObjectBehavior: DeleteAll
+          object-templates:
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: apps/v1
+                kind: Deployment
```

Comment:
After thinking about this, it might be nice to choose something that doesn't create a container since we are adding load to Kubernetes that doesn't relate to the Config Policy controller resource utilization.

Could you pick something else such as a largish ConfigMap, Secret, or something else?